client/src/utils/filterConversion.js

```diff
@@ -10,41 +10,53 @@
  * Use quotations (') around values containing spaces.
  */
 
+/** Default filters are set, unless explicitly specified by the user. */
+export const defaultFilters = {
+    deleted: false,
+    visible: true,
+};
+
+/** Add comparison aliases i.e. '*>value' is converted to '*_gt=value' */
+export const validAlias = [
+    [">", "_gt"],
+    ["<", "_lt"],
+];
+
 /** Converts user input to backend compatible date. */
-function toDate(value) {
+const toDate = (value) => {
     return Date.parse(value) / 1000;
-}
+};
 
 /** Converts user input for case-insensitive filtering. */
-function toLower(value) {
+export const toLower = (value) => {
```

Comment:
Talked about this, we intend to restore the `function NAME() {}` notation here. More details will follow separately. We intend to add this and other conventions to the docs for clarificiation but thats not part of this PR.